src/app/zap-templates/zcl/data-model/chip/webrtc-requestor-cluster.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/WebRTC_Requestor.adoc
+Parameters: in-progress 
+Git: 0.9-fall2024-228-g54b1ae035
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>Cameras</domain>
+    <name>WebRTC Transport Requestor</name>
+    <code>0x0554</code>
+    <define>WEBRTC_TRANSPORT_REQUESTOR_CLUSTER</define>
+    <description>The WebRTC transport requestor cluster provides a way for stream consumers (e.g. Matter Stream Viewer) to establish a WebRTC connection with a stream provider.</description>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="CURRENT_SESSIONS" type="array" entryType="WebRTCSessionStruct" default="0">CurrentSessions</attribute>
+    <command code="0x01" source="client" name="Offer" optional="false">
```

Comment:
We need support in Alchemy for Large message quality tag in xml for commands.
Filed https://github.com/project-chip/alchemy/issues/11.

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/35951 top add this later.